stylesheets/components/slider.css

```diff
@@ -0,0 +1,15 @@
+
+
+.slider-container{
+     margin-top: 10rem;
+     margin: 5rem;
```

Comment:
you should put margin: 5rem, above margin-top: 10rem; so that top margin does not get overridden to 5rem